File: utils/variables.py
import os
import tomllib


PROJECT_NAME = "sandwich"
AUTHOR = "sysnefo"
USER = os.environ.get("USER")
VERSION = "0.0.1"  # major.minor.fix
CONFIG_PATH = f"/home/{USER}/.config/{AUTHOR}/{PROJECT_NAME}"
DATABASE_PATH = f"/home/{USER}/.local/share/{AUTHOR}/{PROJECT_NAME}"
DATABASE_NAME = "sandwich_data"
PORT = 5000

File: utils/variables.go
package utils

import (
	"fmt"
	"log"
	"os"
)

var Conf Config
var USER = os.Getenv("USER")
var PROJECT_NAME = "sandwich"
var CONFIG_LOCATION = fmt.Sprintf("/home/%s/.config/%s/", USER, PROJECT_NAME)

func InitializeVariables() {
	c, err := LoadConfig(CONFIG_LOCATION)
	if err != nil {
		log.Fatal(err)
	}
	Conf = c
}
